features/gba-dev/uitsluiting-kiesrecht.feature

```diff
@@ -0,0 +1,20 @@
+# language: nl
+
+Functionaliteit: Uitsluiting kiesrecht
+
+
+	@gba
+	Abstract Scenario: Geen aanduiding uitgesloten kiesrecht (38.10) wel een einddatum uitsluiting kiesrecht
+	  Gegeven het systeem heeft een persoon met de volgende gegevens
```

Comment:
Bij de historie api gebruiken we nu de volgende gegeven. Willen we die hier ook gebruiken?

```
	  Gegeven de persoon met burgerservicenummer '000000012' heeft de volgende 'kiesrecht' gegevens
```

Reply:
gewijzigd zoals voorgesteld